data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/RestApiUtils.java

```diff
@@ -36,4 +36,11 @@ public static String validateQueryParameter(
     }
     throw new IllegalArgumentException(String.format("'%s' cannot be null or empty.", key));
   }
+
+  public static int getParameterValueAsInt(
+      final Map<String, List<String>> parameterMap, final String key)
+      throws IllegalArgumentException {
+    String stringValue = validateQueryParameter(parameterMap, key);
+    return Integer.valueOf(stringValue);
```

Comment:
this could throw a numberFormatException at integer.valueOf if we aren't passing through an integer... in that case ideally we'll see BAD_ARGUMENT..